tests/integration_tests/security/demo_seccomp/src/bin/demo_jailer.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we want a demo jailer that exerts the default seccomp rules as well?

Reply:
The demo jailer can take any rules as input. The question then becomes: do we want a test that installs the default seccomp rules.

And this it a tricky question. Esentially, the Firecracker process is a jailer that installs the default seccomp rules.

Do you have any specific idea of a test scenario that would use such a jailer?